documentation/feature/weaponselect.md

```diff
@@ -5,30 +5,41 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Weapon select
-The number key can be used to quickly switch between weapons. (1 key - pistol, 2 key - rifle, 3 key - grenade launcher, 4 key - rocket launcher, 5 key - binocular)
+### 1.1 Holster weapon
+Adds the ability to holster a weapon on the back.
 
-### Holster weapon
-Adds the ability to holster a weapon on the back. (0 key)
+### 1.2 Quick weapon select
+Adds key bindings to quickly switch weapons while on foot or in a vehicle. (Unbound by default).
```

Comment:
"...on foot or in a vehicle (unbound by default)."
I think there's better ways to write this sentence.. Think about it. :p